components/Explorer/ExplorerPluginDetails.tsx

```
@@ -62,6 +63,53 @@ export function ExplorerPluginDetails({ plugins, type }: { plugins: PluginsList
+              <LabelTitle>Raw Bytes</LabelTitle>
+              <Text fz="sm" style={{ wordBreak: 'break-all', fontFamily: 'monospace' }}>
+                {appData.data instanceof Uint8Array
+                  ? Array.from(appData.data)
+                    .map(b => b.toString(16).padStart(2, '0'))
+                    .join(' ')
+                  : appData.data}
+              </Text>
+            </div>
+          </Stack>
+        </Fieldset>
+      ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)